dbms/src/IO/Compression/CompressionCodecLightweight.cpp

```diff
@@ -69,12 +69,13 @@ UInt32 CompressionCodecLightweight::doCompressData(const char * source, UInt32 s
     case CompressionDataType::Float32:
     case CompressionDataType::Float64:
     case CompressionDataType::String:
+    case CompressionDataType::Unknown:
```

Comment:
Is it ok to do compression on "unknown" data type?

Reply:
yes, I add "unknown" to distinguish between string (char/varchar) and other types (binary).